ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/WorkloadCves.selectors.js

```diff
@@ -55,6 +55,7 @@ export const selectors = {
     paginationNext: "button[aria-label='Go to next page']",
     severityIcon: (severity) => `span.pf-v5-c-icon:contains('${severity}')`,
     loadingSpinner: 'svg[role="progressbar"][aria-valuetext="Loading..."]',
+    namespaceViewButton: 'button:contains("Prioritize by namespace view")',
```

Comment:
An question, not necessarily a change request, about pro and con of selector when a helper function encapsulates access to the DOM element.

Reply:
Fair point, and I don't see any reason we would ever need this selector other than via use in that helper function.